phalcon/mvc/collection.zep

```diff
@@ -675,6 +675,37 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
 		return !count(messages);
 	}
 
+	/**
+	 * Check whether validation process has generated any messages
+	 *
+	 *<code>
+	 * use Phalcon\Mvc\Model\Validator\ExclusionIn as ExclusionIn;
+	 *
+	 * class Subscriptors extends \Phalcon\Mvc\Collection
+	 * {
+	 *     public function validation()
+	 *     {
+	 *         $this->validate(
+	 *             new ExclusionIn(
+	 *                 [
+	 *                     "field"  => "status",
+	 *                     "domain" => ["A", "I"],
+	 *                 ]
+	 *             )
+	 *         );
+	 *
+	 *         if ($this->validationHasFailed() == true) {
```

Comment:
```suggestion
	 *         if ($this->validationHasFailed() === true) {
```

OR

```suggestion
	 *         if ($this->validationHasFailed()) {
```

Reply:
I would prefer actually the yoda one

`if (true === $this->validationHasFailed()) {`